packages/test-interface-criteria/src/criteria/internal/InternalCriteriaFutureScheduledAndDoesNotRequireDetails.js

```diff
@@ -0,0 +1,45 @@
+const {
+  createCriteria,
+  mustNotRequireAttendeeDetails,
+  mustNotRequireAdditionalDetails,
+} = require('../criteriaUtils');
+const { openBookingFlowRequirementArrayConstraint } = require('../../testDataShape');
+const { InternalCriteriaFutureScheduledOpportunity } = require('./InternalCriteriaFutureScheduledOpportunity');
+
+/**
+ * An opportunity which occurs in the future and does not require additional/attendee details
+ *
+ * This shouldn't be used for any tests, as it is not an [official criteria](https://openactive.io/test-interface/).
+ * It's just a useful basis for other criteria to include constraints from.
+ */
+const InternalCriteriaFutureScheduledAndDoesNotRequireDetails = createCriteria({
+  name: '_InternalCriteriaFutureScheduledAndDoesNotRequireDetails',
+  opportunityConstraints: [],
+  offerConstraints: [
+    // TODO why is the attendee details constraint in this criteria ..?
```

Comment:
I've removed this now obsolete comment. Will not commit until merge time so as not to have to wait for another CI pass 😧